checkov/sca_package_2/output.py

```diff
@@ -86,20 +86,22 @@ def create_cli_output(fixable: bool = True, *cve_records: list[Record]) -> str:
             #  this shouldn't happen
             logging.error(f"'vulnerability_details' is not set for {record.check_id}")
             continue
+        package_alias = get_package_alias(
+            record.vulnerability_details.get("root_package_name", record.vulnerability_details["package_name"]),
+            record.vulnerability_details.get("root_package_version", record.vulnerability_details["package_version"])
+        )
```

Comment:
1. maybe rename 'package_alias' to 'root_package_alias'
this way it will be a little bit clearer that we are grouping by the root packages.
2. is it possible that root_package_name is empty? what happen in case of the package is root ? I think that both package_name and root_package_name should be not empty and equal (the same for package_version). If I'm right, is is enough to have
```
record.vulnerability_details.get["root_package_name"],
```
unless I'm missing anything

Reply:
@itai1357 
1. sure, i'll rename it
2. in case the cve is on a package which is a root so we'll have the same value in "package_name" and "root_package_name", and the same value for "package_version" and "root_package_version". we are using this fact in the creation of the cve table lines. so "root_package_name" or "root_package_version" won't be empty anyway